### PR TITLE
Updated some code/cmd running lines without style

### PR DESCRIPTION
**Pull request by brunoocasali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,13 +91,13 @@ specific RuboCop release.**
 
 **RuboCop**'s installation is pretty standard:
 
-```
+```sh
```
Comment (bbatsov) [inline README.md]: Isn't `sh` for highlighting `sh` scripts? To my knowledge there's nothing for our exact situation (although I've seen many people use `sh` and `bash` for this).

Comment (awwaiid) [inline README.md]: I think 'sh' and individual command line shell commands should be pretty much the same thing.

Somewhat tangentally, I'm not a fan of all the "$" prompt prefixes, as they hinder copy/paste :)

Comment (bbatsov) [inline README.md]: True that. But on the other hand they make it crystal clear that something is a shell command.

Comment (awwaiid) [inline README.md]: I think it's a sensible change.

Comment (brunoocasali) [inline README.md]: So, what do you think to change to bash key highlight ? Or keep in this form? :D

Comment (bbatsov) [inline README.md]: It's all the same to me.

**Comment by alexdowad** [comment]
This looks good.

**Comment by bbatsov** [comment]
Totally forgot about this PR. Let's merge it! :-)

**Pull request merged**